Source/Core/VideoCommon/Present.cpp

```diff
@@ -244,9 +280,12 @@ Presenter::ConvertStereoRectangle(const MathUtil::Rectangle<int>& rc) const
   return std::make_tuple(left_rc, right_rc);
 }
 
-float Presenter::CalculateDrawAspectRatio() const
+float Presenter::CalculateDrawAspectRatio(bool allow_stretch) const
```

Comment:
I think you could just pass in the `AspectMode` instead, avoiding the below if check.

Reply:
Is there a reason this was resolved?

Reply:
*Comment went lost for some reason.
I think this is an implementation detail, both versions are fine, I just opted to "add" to what was already there instead of refactoring how the `AspectMode`  is carried around functions. I also personally don't really like the idea of passing in a fake `AspectMode` edited from outside (removing stretch from it), I prefer passing in a separate modifier value for it, so each function can do as they prefer.

Reply:
Sure it's an implementation detail but that's what code reviews are all about :) .  Anyway, I agree it's minor, just seems wrong to do the same work in two or three places.